cluster-autoscaler/cloudprovider/clusterapi/clusterapi_controller.go

```diff
@@ -24,9 +24,9 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	kubeinformers "k8s.io/client-go/informers"
 	"k8s.io/client-go/tools/cache"
-	"sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+	"sigs.k8s.io/cluster-api/pkg/apis/machine/v1beta1"
```

Comment:
this should be github.com/openshift/clsuter-api

Reply:
Fixed in 2c6b5ed and e53ed54